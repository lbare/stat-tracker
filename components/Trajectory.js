import { Text } from 'react-native';

const Trajectory = () => {
  return (
    <>
      <Text className='text-30 text-center text-black'>TRAJECTORY</Text>
    </>
  );
};

export default Trajectory;
