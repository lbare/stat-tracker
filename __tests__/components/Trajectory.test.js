import React from 'react';
import renderer from 'react-test-renderer';

import Trajectory from '../../components/Trajectory';

describe('<Trajectory />', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Trajectory />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
