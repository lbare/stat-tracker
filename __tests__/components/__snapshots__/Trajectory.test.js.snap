// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Trajectory /> renders correctly 1`] = `
<Text
  style={
    [
      {
        "textAlign": "center",
      },
      {
        "color": "#000",
      },
    ]
  }
>
  TRAJECTORY
</Text>
`;
